app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -17,13 +24,43 @@
 
 <p> If you’re interested in teaching a different subject, you can still <a href="/train-to-be-a-teacher/get-school-experience">get school experience</a> and find out what it’s like to be in a classroom.</p>
 
+<section class="col col-640">
+    <%= render Content::QuoteComponent.new(
+      text: "Towards the end of the internship, I was able to teach parts of lessons that I had planned, and I really started to find my feet at the front of the classroom.",
+      name: "Ben, former teaching intern",
+      background: "grey",
+      large: false
+    ) %>
+  </section>
+
 <h2> What you'll cover</h2>
 
 <p>You'll get a feel for school life, which could include:</p>
   <ul>
-     <li>classroom experience, which could include helping with lessons and providing support to pupils</li>
+     <li>classroom experience, helping with lessons and providing support to pupils</li>
```

Comment:
is classroom experience maybe not quite descriptive enough? is it all classroom experience, really?

I wonder if you could ditch this first sentence?
```suggestion
```